### PR TITLE
Feature/okx marketplaces

### PR DESCRIPTION
**Pull request by zhengyuhui94**
OKX NFT marketpalce access dappradar marketplaces

### CODE REVIEW

**Review by XuXiaoGH — COMMENTED** [review]
Passed

File: src/adapters/okx-avalanche-1/index.ts [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    range: number;
+    chunkSize: number;
+    sdk: any;
+
+    constructor() {
+        this.name = "okx-avalanche-1";
+        this.protocol = "avalanche";
+        this.block = 26572369;
+        this.deprecatedAtBlock = 26572425;
+        this.contract = "0xA88BA7EA2912cde5d632C8De048A17f2ea3e75A0";
```
Comment (liutaurasjanuskas) [inline src/adapters/okx-avalanche-1/index.ts:31]: Please lowercase contract address

File: src/adapters/okx-bsc-1/index.ts [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    range: number;
+    chunkSize: number;
+    sdk: any;
+
+    constructor() {
+        this.name = "okx-bsc-1";
+        this.protocol = "binance-smart-chain";
+        this.block = 25862299;
+        // this.deprecatedAtBlock = 16625257;
+        this.contract = "0xcCE3E3F79cf9091386F84610bb06947E2fc232A3";
```
Comment (liutaurasjanuskas) [inline src/adapters/okx-bsc-1/index.ts:31]: Please lowercase contract address

File: src/adapters/okx-ethereum-1/index.ts [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+    range: number;
+    chunkSize: number;
+    sdk: any;
+
+    constructor() {
+        this.name = "okx-ethereum-1";
+        this.protocol = "ethereum";
+        this.block = 16681307;
+        // this.deprecatedAtBlock = 16625257;
+        this.contract = "0x92701D42E1504ef9FCe6d66A2054218b048ddA43";
```
Comment (liutaurasjanuskas) [inline src/adapters/okx-ethereum-1/index.ts:31]: Please lowercase contract address

File: src/adapters/okx-polygon-1/index.ts [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+    range: number;
+    chunkSize: number;
+    sdk: any;
+
+    constructor() {
+        this.name = "okx-polygon-1";
+        this.protocol = "polygon";
+        this.block = 39574716;
+        // this.deprecatedAtBlock = 39539879;
+        this.contract = "0x954dab8830aD2B9C312Bb87aCe96f6Cce0F51E3a";
```
Comment (liutaurasjanuskas) [inline src/adapters/okx-polygon-1/index.ts:31]: Please lowercase contract address

File: src/adapters/okx-avalanche-1/index.ts [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+    pathToAbi: string;
+    range: number;
+    chunkSize: number;
+    sdk: any;
+
+    constructor() {
+        this.name = "okx-avalanche-1";
+        this.protocol = "avalanche";
+        this.block = 26572369;
+        this.deprecatedAtBlock = 26572425;
```
Comment (liutaurasjanuskas) [inline src/adapters/okx-avalanche-1/index.ts:30]: Is this `deprecatedAtBlock` needed?

**Review by liutaurasjanuskas — APPROVED** [review]

**Pull request merged**